README.md

```diff
@@ -17,14 +17,16 @@ In this tutorial, you will be integrating a fitting room that was already config
 * a link to your personal fitting room
 * the list of model IDs that will be tried on (you will use it to select which models to display and which one to start with)
 
-Virtual Fitting Room could only be comfortably used [on a smartphone](#mobile-show-the-fitting-room). You probably already have different versions of your website for desktop and mobile devices, but if not, you will need to implement the selection on your side. For [desktop version](#desktop-display-a-qr-code) we recommend displaying a QR code with the link to smoothly guide the user to the try-on page.
+Virtual Fitting Room could only be comfortably used for virtual try-on [on a smartphone](#mobile-show-the-fitting-room). You probably already have different versions of your website for desktop and mobile devices, but if not, you will need to implement the selection on your side. For [desktop version](#desktop-display-a-qr-code) we recommend displaying a QR code with the link to smoothly guide the user to the virtual try-on page.
+
+The 3D Viewer component may be displayed on both desktop and mobile devices.
 
 ## Mobile: show the fitting room
 
-The essential part of integration is simply to load Virtual Fitting Room into an iframe on the page or just open it in a separate page. Choose the look and user experience you would prefer. There are three main options:
+The essential part of integration is simply to load the Virtual Fitting Room or 3D Viewer component into an iframe on the page or just open it in a separate page. Choose the look and user experience you would prefer. There are three main options:
```

Comment:
We can keep Virtual Fitting Room here. We use it as a generic name for the solution that can present either virtual tey-on or 3D Viewer